R/make-objects.R

```diff
@@ -243,11 +243,16 @@ make_exemplar_model_from_data = function(
     group = NULL,
     category = "category",
     cues,
-    sim_function = function(x, y) {
-      j <- 2
-      k <- 1
-      distance <- (x - y)^j
+    sim_function = function(x, y, j = 2, k = 1) {
+      # If j = 1, this uses Manhattan city block distances
+      # If j = 2, this uses Euclidean distances
+      distance <- sum((x - y)^j)^-j
```

Comment:
just checking: this expression gives 1/(sum of distances raised by j)

Reply:
I meant line 249

Reply:
Good question. Not quite. What you wrote would be sum((x-y)^j)^-1.

But I made mistake. What I want is the generalization of the formula for the Euclidean distance:

sqrt(sum((x - y)^j)) = sum((x - y)^j)^(1/2)

So, it should be:

sum((x - y)^j)^(1/j). I'll push a fix. Can you look over it again and then approve if all looks good?